### ISSUE TITLE
[DataCap Application] Sequence Read Archive

### ISSUE THREAD

**Issue by CC0Nala** [CloudX-Lab/filecion#10]
### Data Owner Name

National Library of Medicine (NLM)

### Data Owner Country/Region

United States

### Data Owner Industry

Resources, Agriculture & Fisheries

### Website

https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/

### Social Media Handle

https://x.com/NLM_NIH

### Social Media Type

Twitter

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

10PiB

### Expected size of single dataset (one copy)

2.3PiB

### Number of replicas to store

6

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1ws375yxs2pedch4aalvi7rsedc25odfi4sdlyoa

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The Sequence Read Archive (SRA), produced by the National Center for Biotechnology Information (NCBI) at the National Library of Medicine (NLM) at the National Institutes of Health (NIH), stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community's efforts to enhance reproducibility and make new discoveries by comparing data sets.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Buckets in this registry contain public SRA data in the original (user submitted) format from select high value and newly-released studies as well as all public-access SRA formatted ETL+BQS data. Also included is all SRA metadata that can be leveraged for attribute-based data discovery.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Hong Kong

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We used either Lotus or Boost way to prepared data. It depends on the way sps expect to receive the data.
```

### Please share a sample of the data

```text
s3://sra-pub-run-odp/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

More than 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Filmine, Big Data Exchange, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f01344987|HongKong
f02029743|Singapore
f02114994|Sichuan
f01844043|US
f02207907|US
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by CC0Nala** [comment]
Hello allocator @CloudX-Lab 

Can you approve my request?

Thank you

**Comment by CloudX-Lab** [comment]
@CC0Nala Hello, please send an email to chenhuanyu1220@gmail.com for comfirmation.

And answer questions as elow:

Why do you want to save data in Filecoin?
Where do you want to choose sp?
Is your behavior legal in your country/region?
Do you agree to publicly disclose trading information after the application is approved?

**Comment by CC0Nala** [comment]
@CloudX-Lab Dear allocator, we have sent an email to you.

Fil+ is a project to develop storage. I think it's safe and stable to store data in the filecoin network.
We'll find sp from HongKong, america, singapore or more countries/regions.
Yes.
Yes.

**Comment by CloudX-Lab** [comment]
@CC0Nala ![image](https://github.com/CloudX-Lab/filecion/assets/142370129/1ac9aec4-9c43-483b-97fa-92bd7c120998)
ok. I got your email and confirm your identity. We'd like to support you. Are you ready for anything?

**Comment by CC0Nala** [comment]
@CloudX-Lab Yes! We have contacted sps and they also prepared enough tokens to take deals. Please approve us.

**Comment by CloudX-Lab** [comment]
checker:manualTrigger

**Comment by CloudX-Lab** [comment]
Decided to support next round and continue to follow the usage data.

**Comment by CC0Nala** [comment]
@CloudX-Lab Dear Allocator, Can you please allocate next round for us?

**Comment by CloudX-Lab** [comment]
checker:manualTrigger

**Comment by CloudX-Lab** [comment]
@CC0Nala Please let all your SPs support retrieval. I saw your progress and glad to support you. Keep and continue approve your successful retrieval result.

**Comment by CC0Nala** [comment]
ok sir. Most of sps have already support retrieval. We have stopped cooperating with those sps who failed to support retrieval. Thank you.

**Comment by CC0Nala** [comment]
@CloudX-Lab Hello sir, can you allocate the next round?

Here's our updated sp list. Please support us.
f03159626 | Dulles Town Center
f03087446 | Los Angeles
f03147753 | New York City
f03136895 | Dulles Town Center
f03144188 | Dulles Town Center
f03090976 | Dulles Town Center
f03089826 | Dulles Town Center
f03157879 | Los Angeles

**Comment by CloudX-Lab** [comment]
@CC0Nala Good, I see high percentage of successful retrievals in the report. We can give you this round support.

**Comment by CC0Nala** [comment]
@CloudX-Lab Thank you sir!

**Comment by CloudX-Lab** [comment]
> Client used 75% of the allocated DataCap. Consider allocating next tranche.

It seems the bot has some bugs.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by filecoin-watchdog** [comment]
> > Client used 75% of the allocated DataCap. Consider allocating next tranche.
> 
> It seems the bot has some bugs.

Why do you think so? It looks like 3 PiB out of granted 4 are used, which is exactly 75% of the DC used.

**Comment by CloudX-Lab** [comment]
checker:manualTrigger
